third_party/2and3/yaml/__init__.pyi

```diff
@@ -1,10 +1,10 @@
 from typing import Any
-# from yaml.error import *
-# from yaml.tokens import *
-# from yaml.events import *
-# from yaml.nodes import *
-# from yaml.loader import *
-# from yaml.dumper import *
+from yaml.error import *
```

Comment:
Unfortunately that doesn't help, from the sound of it flake8 doesn't like wildcard imports. Other ideas?